src/index.html

```diff
@@ -7,12 +7,71 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
+
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__product product">
+        <img
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+          class="product__image-iMac"
+        />
+
+        <a
+          href="#"
+          class="product__name"
+        >
+          APPLE A1419 iMac 27&#34; Retina 5K Monoblock (MNED2UA/A)
+        </a>
+
+        <span class="product__code">Product code: 195434</span>
+
+        <div class="product__review review">
+          <div class="review__stars">
+            <span class="star review__star"></span>
+            <span class="star review__star"></span>
+            <span class="star review__star"></span>
+            <span class="star review__star"></span>
+            <span class="star review__star"></span>
```

Comment:
Fix classes and add modifier class
```suggestion
          <div class="stars stars--4">
            <span class="stars__star"></span>
            <span class="stars__star"></span>
            <span class="stars__star"></span>
            <span class="stars__star"></span>
            <span class="stars__star"></span>
```